sdks/go/pkg/beam/core/runtime/graphx/coder.go

```diff
@@ -50,6 +50,22 @@ const (
 	urnCoGBKList   = "beam:go:coder:cogbklist:v1" // CoGBK representation. Not a coder.
 )
 
+func KnownStandardCoders() []string {
```

Comment:
Since this lives in the same package as the caller, we can make the function un-exported (lowercase k for knownStandardCoders() instead).

It's reasonably to have this function live here, due to locality with the coder URNs